flink-tests/src/test/java/org/apache/flink/test/iterative/aggregators/AggregatorsITCase.java

```diff
@@ -199,12 +201,17 @@ public void testConvergenceCriterionWithParameterForIterate() throws Exception {
 				new NegativeElementsConvergenceCriterionWithParam(3));
 
 		DataSet<Integer> updatedDs = iteration.map(new SubtractOneMap());
-		iteration.closeWith(updatedDs).writeAsText(resultPath);
-		env.execute();
+		List<Integer> result = iteration.closeWith(updatedDs).collect();
+		Collections.sort(result);
+
+		List<Integer> expected = new ArrayList<Integer>() {{
+			add(-3); add(-2); add(-2); add(-1); add(-1);
+			add(-1); add(0); add(0); add(0); add(0);
+			add(1); add(1); add(1); add(1); add(1);
+		}};
+		Collections.sort(expected);
```

Comment:
let's remove this so the port remains straight-forward. I don't wanna worry about these sorts changing the semantics of the test.